README.md

```diff
@@ -20,6 +20,7 @@ Command-line tools for Ringpop
     status       Status of members in ring
     partitions   Show partition information of a ring
     top          General membership information
+    reap         Remove the faulty nodes from this nodes membership
```

Comment:
Sorry I was confused that this PR contained the reaping also, but after reading the description of the PR I understood why reap was here. I think the name is fine